src/sidebar/components/hooks/test/use-user-filter-options-test.js

```diff
@@ -0,0 +1,149 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import useUserFilterOptions from '../use-user-filter-options';
```

Comment:
I've been coming around to the view recently that default exports should be avoided in most cases. The primary reason is because it tends to increase the chance that a function/class is unnecessarily given different names in different files. On that basis I've recently avoided it except for UI components where there is a strong convention for the association of file and class names.

Reply:
OK, I updated to no `default` export. I also renamed the module to `use-filter-options` because it's plausible we might end up putting more than one related hook here, too.